ui/apos/components/AposExportModal.vue

```diff
@@ -229,6 +237,9 @@ export default {
             type: this.type
           }
         });
+        this.checkedRelatedTypes = this.relatedTypes;
+        const height = this.checkedRelatedTypes.length ? this.checkedRelatedTypes.length * 24 + 95 : 120;
```

Comment:
24, 95, 120: what are they corresponding to?
maybe a constant would be good

Reply:
Ok